package.json

```diff
@@ -37,6 +38,14 @@
         "$ fix.es5"
       ]
     },
+    "umd": [
+      [
+        "echo \"(function(A,G){if(typeof define=='function'&&define.amd)define([],G);else",
+        "if(typeof module=='object'&&module.exports)module.exports=G();else",
+        "A.HyperHTMLElement=G()}(typeof self!='undefined'?self:this,function(){\">umd.js;cat",
+        "es5.min.js>>umd.js;echo \"return HyperHTMLElement}));\">>umd.js"
```

Comment:
Yes, and if you copied and paste that, you'd have `min.js>>umd.js`, not `es5.min.js>>umd.js`.

Why did you umd the transpiled, most problematic, version instead of the regular one?

If you want the umd for the es5 then we need two umd, as it is for the rest: `umd.js` and `es5.umd.js` (or `umd.es5.js`),